yarn-project/end-to-end/src/e2e_fees/fees_test.ts

```diff
@@ -185,16 +185,6 @@ export class FeesTest {
         const canonicalFeeJuice = await getCanonicalFeeJuice();
         this.feeJuiceContract = await FeeJuiceContract.at(canonicalFeeJuice.address, this.aliceWallet);
         this.coinbase = EthAddress.random();
-
-        this.feeJuiceBridgeTestHarness = await FeeJuicePortalTestingHarnessFactory.create({
```

Comment:
What's this?

Reply:
Cleanup. The benchmarks are based off these tests and I realized that was unnecessary.